lib/instrumentation/elasticsearch-shared.js

```diff
@@ -10,64 +10,50 @@
 // - elasticsearch - the legacy Elasticsearch JS client
 // - @elastic/elasticsearch - the new Elasticsearch JS client
 
-const querystring = require('querystring')
```

Comment:
👍 dings the "two `require('querystring')` references left before we can drop another dep." bell 🛎️